scripts/build-graphql-schema.ts

```diff
@@ -4,7 +4,7 @@ import { GraphQLDefinitionsFactory } from '@nestjs/graphql'
 import { createApp } from '@island.is/infra-nest-server'
 
 const argv = yargs(process.argv.slice(2))
-const args = argv.string('rootModule').help().argv
+const args = argv.string('rootModule').string('_').help().argv
```

Comment:
Should still be only a single argument. Shouldn't you rather replace the `rootModule` with `_`.

Reply:
Added a command and a stack trace in the PR description. 
Yes seems that the `rootModule` is not used, so updated it to be a single `.string()` function with `_`